api/client/clientModel.js

```diff
@@ -13,7 +13,11 @@ const findAll = () => {
 };
 
 const findById = async (id) => {
-  return db('clients').where({ id }).first();
+  const client = db('clients').where({ id }).first();
```

Comment:
this db call returns a promise, is there a reason you didn't await the response here?